analytics.go

```diff
@@ -416,9 +401,20 @@ func (c *client) maxBatchBytes() int {
 	return maxBatchBytes - len(b)
 }
 
+func (c *client) reportMetrics(name string, value float64, tags []string) {
+	if c.DDStatsdClient == nil {
+		return
+	}
+
+	err := c.Config.DDStatsdClient.Gauge("submitted.success", value, tags, 1)
```

Comment:
c.Config.DDStatsdClient => c.DDStatsdClient?

Also, these are counters, not gauge, so float64=>int64, and Gauge=>Count.

Difference is that gauge is the value reported at time stamp (e.g. CPU usage) while counter is number of occurrences within time period.  Counters are aggregated when zoom out time axis, gauges are not.